terraform/slurm_cluster/main.tf

```diff
@@ -55,7 +55,7 @@ locals {
 
 module "bucket" {
   source  = "terraform-google-modules/cloud-storage/google"
-  version = "~> 3.0"
+  version = "~> 5.0"
```

Comment:
Noting for record that 6.0 come out literally 2 hours ago but _requires_ TPG 5.x. I prefer we stay at cloud-storage 5.x.

Reply:
No action needed. Marking resolved.